.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+  // Show the path in the top window bar.
+  "window.title": "${rootName}${separator}${activeEditorMedium}",
+
+  // Formatting
+  "editor.formatOnSave": true,
```

Comment:
there is some beautify excludes that we have in others like prettier-poet, can you copy those over here?

Reply:
Hm, I can't find any `beautify.ignore` setting in VSCode with my current configuration. I assume it's a setting for a different extension is not in the list of recommended extensions, but that, if installed separately, would interfere with ESLint auto-formatting? Is that correct?

Reply:
yeah, with the prettier auto-formatting. 

We want this:
```
"beautify.ignore": ["**/*.js", "**/*.md"],
```

Reply:
Eh, I'll just add that to your PR.